src/app/features/reports/components/report-picker/report-picker.component.html

```diff
@@ -0,0 +1,10 @@
+<mat-menu #rootMenu="matMenu">
+  <button *ngFor="let report of getReportValues()"
+          mat-menu-item
+          (click)="chooseReport.emit(report)">{{report}}</button>
```

Comment:
I see new approach here, though I'm not sure if it will be easy to test, time or Ziela will show.